lab_share_lib/processing/rabbit_message_processor.py

```diff
@@ -68,6 +70,12 @@ def process_message(self, headers, body):
             LOGGER.error("RabbitMQ message received containing multiple AVRO encoded messages.")
             return False  # Send the message to dead letters.
```

Comment:
Is this going to send all messages involved to dead letters or the first in this case?

Reply:
All.  We agreed a couple years ago that we wouldn't support multiple objects in the same message.  The wording of the log may be a bit misleading as we've conflated the RabbitMQ message with the objects that are encoded within it.  So what you are sending to dead letters is the (single) RabbitMQ message that had inside it an array of multiple objects.